A32NX/html_ui/Pages/A32NX_Core/A32NX_Warning.js

```diff
@@ -0,0 +1,78 @@
+class A32NX_Warning {
+    constructor(_core) {
+        console.log('A32NX_Warning constructed');
+
+        this.warningLeft = false;
+        this.warningRight = false;
+        this.cautionLeft = false;
+        this.cautionRight = false;
+        this.AutothrottleWarningCanceled = false;
+        this.athrdeltaTime = 0;
+        this.apdeltaTime = 0;
```

Comment:
You can use logic nodes from A32NX_Logic for this. Look at the FWC for examples on how they work.